Makefile.arduino

```diff
@@ -14,7 +15,7 @@
 			LWIP_LIB=liblwip$$((2+4*v6))-$$mss$$featname.a \
 			LWIP_LIB_RELEASE=../../lib/liblwip$$((2+4*v6))-$$mss$$featname.a \
 			LWIP_INCLUDES_RELEASE=../include \
-			TOOLS=../../../xtensa-lx106-elf/bin/xtensa-lx106-elf- \
+			TOOLS=$(abspath $(wildcard $(PREFIX)))/bin/xtensa-lx106-elf- \
```

Comment:
$(abspath ...) works with most paths. Only addition here is $(wildcard ...), since make would not expand tilde like in my example